app/controllers/documents_controller.rb

```diff
@@ -70,7 +70,8 @@ def update_verification_type
     family_member = FamilyMember.find(params[:family_member_id]) if params[:family_member_id].present?
     reasons_list = VlpDocument::VERIFICATION_REASONS + VlpDocument::ALL_TYPES_REJECT_REASONS + VlpDocument::CITIZEN_IMMIGR_TYPE_ADD_REASONS
     if (reasons_list).include? (update_reason)
-      verification_result = @person.consumer_role.admin_verification_action(admin_action, @verification_type, update_reason)
+      admin_rejected = format_string_to_bool(params[:admin_rejected])
+      verification_result = @person.consumer_role.admin_verification_action(admin_action, @verification_type, update_reason, admin_rejected)
```

Comment:
I don't think we need this flag `admin_rejected`. you can just work with `admin_action` - you need to set state as `rejected` when `admin_action` is return for def